static.config.js

```diff
@@ -725,8 +743,21 @@ const makeAllPages = async (langCode, incrementalPageId) => {
   let allPages = await Promise.all(
     pages.map(pageAtUrlInfo => {
       if (!!pageAtUrlInfo.node.janisInstances.length) {
+        return Promise.all(
+          pageAtUrlInfo.node.janisInstances.map(instanceOfPage =>
+            buildPageAtUrl(
+              pageAtUrlInfo.node,
+              instanceOfPage,
+              client,
+              pagesOfGuidesData,
+            ),
+          ),
+        );
+      }
+
+      // not all pages have instances (events and locations not under departments)
```

Comment:
hmm I wasn't sure if we were using janisUrls anymore (on Janis). Could events and locations have janisInstances too to keep things consistent? Not a requirement for this PR, just a thought.

Reply:
I did this a few weeks ago: https://github.com/cityofaustin/joplin/compare/4267-fix-instance but then thought it wasnt necessary (felt like too much copying / pasting on joplin) but do you think i should bring it back? for the sake of janis?